spec/index.html

```diff
@@ -816,6 +816,27 @@ <h3>Content Negotiation of RDF Datasets</h3>
       the consumer is expected to use the <a data-lt="rdf dataset">RDF dataset's</a> default graph.</p>
 
     </section>
+
+    <section id="section-dataset-quad" class="informative">
+      <h3>Dataset as a set of quads</h3>
+
+      <p>A <dfn>quad</dfn> is a conceptual projection of a <a>triple</a> with
+        either the <a>default graph</a> or a <a>named graph</a>.
+        A <a>quad</a> is the generalization of a <a>triple</a>
```

Comment:
This is quite different to #5.

The projection (as in "a view of" - c.f. SQL) is the other way round - a triple is the projection of a quad without the graph name component.

In #5, a quad is a triple and an optional graph name.

Reply:
I also find the phrasing of the first sentence not so suitable. It is not really clear what a "conceptual projection" is.

My definition would be:
```
A <dfn>quad</dfn> is a <a>triple</a> associated with an optional <a>graph name</a> and
is used when referring to triples within an <a>RDF dataset</a>.
```

Using that definition as the first paragraph of the section, as the next paragraph I would put the sentence about "An RDF dataset can be .." (as given in [Andy's comment below](https://github.com/w3c/rdf-concepts/pull/12/files#r1094292891)).

As third paragraph I would then use the following adaptation of the paragraph about quads versus triples.
```
Although a <a>quad</a> without a graph name consists of the same three components as
a <a>triple</a>, it is a distinct concept, as it specifically captures the notion of
a triple within the <a>default graph</a> of an <a>RDF dataset</a>.
```